packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationBilling.tsx

```diff
@@ -1,24 +1,69 @@
 import { useOrganization } from '@clerk/shared/react';
 
+import { ChangePlanButton, ManagePlanScreen, OrganizationPlanCard } from '../../common/Billing';
+import { Box, descriptors, Spinner } from '../../customizables';
 import { useFetch } from '../../hooks';
 
 export const OrganizationBilling = () => {
   const { organization } = useOrganization();
+  const { data: availablePlans, isLoading: isLoadingAvailablePlans } = useFetch(
+    organization?.getAvailablePlans,
+    'availablePlans',
+  );
+  const { data: currentPlan, isLoading: isLoadingCurrentPlan } = useFetch(organization?.getCurrentPlan, 'currentPlan');
 
-  const { data, isLoading } = useFetch(organization?.getAvailablePlans, {});
-  console.log(data, isLoading);
-
-  if (isLoading) {
-    return <div>Loading</div>;
+  if (isLoadingAvailablePlans || isLoadingCurrentPlan) {
+    return (
+      <Box
+        sx={{
+          width: '100%',
+          height: '100%',
+          position: 'relative',
+        }}
+      >
+        <Box
+          sx={{
+            margin: 'auto',
+            position: 'absolute',
+            left: '50%',
+            top: '50%',
+            transform: 'translateY(-50%) translateX(-50%)',
+          }}
+        >
+          <Spinner
+            size='sm'
+            colorScheme='primary'
+            elementDescriptor={descriptors.spinner}
+          />
+        </Box>
+      </Box>
+    );
   }
 
-  if ((data?.data.length || 0) < 1) {
-    return <>No data</>;
-  }
```

Comment:
Couldn't be possible to have 0 available plans ? What will happen if you have a "current" plan but the app owner removes all the entries from the dashboard. Will "available plans" still be populated with the current plan at least ?

Reply:
It isn't possible to have 0 available plans as when we have the Billing enabled there will be always at least one available plan